ext/curl/config.m4

```diff
@@ -1,102 +1,35 @@
 dnl config.m4 for extension curl
 
-PHP_ARG_WITH(curl, for cURL support,
-[  --with-curl[=DIR]         Include cURL support])
+PHP_ARG_ENABLE(curl, for cURL support,
+[  --enable-curl[=DIR]       Include cURL support])
```

Comment:
The `[=DIR]` part here should be dropped, as the option no longer accepts a directory.

Reply:
Ah, whoops.